### PR TITLE
ATC Revamp, Part One Of Unknown

### PR DESCRIPTION
**Pull request by elgeonmb**
In order of descending importance

Adds free traders and MBT to the tracon. Unsure if the lore codex automatically populates from the same definitions but if it does they shouldn't be too out of place.

Bishop now uses the I-independent designation instead of having their own letters.  They're a reletively small boutique cyberware firm, they don't really have enough ships to warrent their own registry numbers.

Gets rid of the "distress beacon" thing.  It might make sense for medical and military response, but it was always kinda weird to have Xion send a priority load of unrefined ore to an emergency beacon.

Expands the missions available to every trans-stellar. I went down the ship naming list and gave everyone anything I thought remotely fit, except for colony ships (which IMO would need their own snowflake code). Morpheus doesn't have MDVs but that's because they may or may not be a hostile partisan. 

New ship names for all the ship lists. Notable changes include Vey-Med having some non-Diona ships and the generic list no longer being single nouns or adjectives.

Gives headquarters to every TSC whose headquarters I know

### CODE REVIEW

**Comment by Atermonera** [comment]
Not your fault on travis, it's found active edges, which is probably the same issue that neerti's lighting optimization ran into for like a month

**Comment by elgeonmb** [comment]
Looks like I accidentally duplicated Xion's entry. Will fix when I am not
on mobile.

**Comment by Belsima** [comment]
i love main battle tanks

**Comment by elgeonmb** [comment]
What

File: code/modules/busy_space/organizations.dm [hunk 1]
```diff
@@ -130,14 +143,14 @@
 	desc = "Hephaestus Industries is the largest supplier of arms, ammunition, and small millitary vehicles in Sol space. \
 	Hephaestus products have a reputation for reliability, and the corporation itself has a noted tendency to stay removed \
 	from corporate politics. They enforce their neutrality with the help of a fairly large asset-protection contingent which \
-	prevents any contracting polities from using their own materiel against them. SolGov itself is one of Hephastus’ largest \
+	prevents any contracting polities from using their own materiel against them. SolGov itself is one of Hephastus' largest \
```
Comment (Atermonera) [inline code/modules/busy_space/organizations.dm]: `Hephaestus`

File: code/modules/busy_space/organizations.dm [hunk 2]
```diff
@@ -207,13 +242,13 @@
 	acronym = "ZH"
 	desc = "Zeng-Hu is an old TSC, based in the Sol system. Until the discovery of Phoron, Zeng-Hu maintained a stranglehold \
 	on the market for medications, and many household names are patentted by Zeng-Hu-- Bicaridyne, Dylovene, Tricordrizine, \
-	and Dexalin all came from a Zeng-Hu medical laboratory. Zeng-Hu’s fortunes have been in decline as Nanotrasen’s near monopoly \
-	on phoron research cuts into their R&D and Vey-Med’s superior medical equipment effectively decimated their own equipment \
+	and Dexalin all came from a Zeng-Hu medical laboratory. Zeng-Hu's fortunes have been in decline as Nanotrasenâ€™s near monopoly \
```
Comment (Atermonera) [inline code/modules/busy_space/organizations.dm]: what

File: code/modules/busy_space/organizations.dm [hunk 3]
```diff
@@ -226,15 +261,15 @@
 	desc = "Ward-Takahashi focuses on the sale of small consumer electronics, with its computers, communicators, \
 	and even mid-class automobiles a fixture of many households. Less famously, Ward-Takahashi also supplies most \
 	of the AI cores on which vital control systems are mounted, and it is this branch of their industry that has \
-	led to their tertiary interest in the development and sale of high-grade AI systems. Ward-Takahashi’s economies \
-	of scale frequently steal market share from Nanotrasen’s high-price products, leading to a bitter rivalry in the \
+	led to their tertiary interest in the development and sale of high-grade AI systems. Ward-Takahashiâ€™s economies \
```
Comment (Atermonera) [inline code/modules/busy_space/organizations.dm]: See above, your apostrophe is fucked up

File: code/modules/busy_space/organizations.dm [hunk 4]
```diff
@@ -246,27 +281,38 @@
...
+		"Curtain"
 		)
 	destination_names = list()
 
 /datum/lore/organization/tsc/bishop
 	name = "Bishop Cybernetics"
 	short_name = "Bishop"
 	acronym = "BC"
-	desc = "Bishop’s focus is on high-class, stylish cybernetics. A favorite among transhumanists (and a bête noire for \
+	desc = "Bishop's focus is on high-class, stylish cybernetics. A favorite among transhumanists (and a bÃªte noire for \
```
Comment (Atermonera) [inline code/modules/busy_space/organizations.dm]: here too

**Pull request merged**